bundles/org.openhab.binding.airq/README.md

```diff
@@ -53,26 +53,28 @@ The rw column is empty if the channel is only readable, w if the channel can be
 | fineDustCnt02_5           | Number:Dimensionless |    | Fine Dust >2,5 µm                                                   |
 | fineDustCnt05             | Number:Dimensionless |    | Fine Dust >5 µm                                                     |
 | fineDustCnt10             | Number:Dimensionless |    | Fine Dust >10 µm                                                    |
-| co                        | Number               |    | CO concentration                                                    |
-| co2                       | Number:Dimensionless |    | CO₂ concentration                                                   |
+| co                        | Number:Density       |    | Carbon monoxide (CO) concentration                                  |
+| co2                       | Number:Dimensionless |    | Carbon dioxide (CO₂) concentration                                  |
```

Comment:
Why is there a difference? I would expect both to be `Number:Density` ?

The other channels are also diffent while they all show the concentration of something.

Reply:
CO2 is measured in ppm like oxygen is measured in %. CO should always be 0%, maybe that's why it's measured in µg/m³. In any case, the difference is correct.

Reply:
Thanks for the details, learned something here ;-)